person.rb

```diff
@@ -24,4 +25,8 @@ def correct_name
   def of_age?
     false unless of_age? >= 18
   end
+
+  def add_rental(date, book)
+    Rental.new(date, book, self)
+  end
```

Comment:
- [ ] Hey there! 👋 Fantastic job on adding the method for adding rentals in the Person class! 🎉 Just a small tweak needed: it seems to be implemented under a private method, but it should actually be a public method. 🚀 Kindly move it out of the private scope to make it accessible where needed. This adjustment ensures that the method is readily available for use and doesn't remain restricted to the private context. 💻✨ Keep up the great work! 🌟